components/reconciler/makefile

```diff
@@ -1,6 +1,6 @@
 OAPI_GENERATOR=oapi-codegen
 OAPI_FILE=./docs/external_api.yaml
-OAPI_REVISION='8e4d9db8e4f2812a966a4f63f3f4b7f75797cf63'
+OAPI_REVISION='7e0f684a7b65f4b4f0b72c8dce44903cd05317ff'
```

Comment:
Replace it with the right revision after merging [this PR](https://github.com/kyma-incubator/reconciler/pull/531)

Reply:
PR is merged, OAPI_Revision can be updated